components/event-publisher-proxy/config/event-publisher-nats/120-peerauthentication.yaml

```diff
@@ -10,4 +10,4 @@ spec:
       app.kubernetes.io/instance: eventing
   mtls:
     mode: "PERMISSIVE"
----
```

Comment:
Is there a good reason to keep it, to not have it STRICT?

Reply:
Deleted it, it was not even used

Reply:
This PA is not used, will be removed